Tests/StreamChatUITests/SnapshotVariant.swift

```diff
@@ -44,3 +48,13 @@ enum SnapshotVariant: String, Hashable, CaseIterable {
         }
     }
 }
+
+extension Array where Element == SnapshotVariant {
+    static var allCases: [SnapshotVariant] {
+        SnapshotVariant.allCases
+    }
+    
+    static var lightAndDarkCases: [SnapshotVariant] {
+        SnapshotVariant.lightAndDarkCases
+    }
+}
```

Comment:
I think these make it easier to use the static properties, instead of using `SnapshotVariant.allCases`, we can simply use `.allCases` in the argument. Also, to add more combinations is also cool IMO.